src/Data/Bit/F2Poly.hs

```diff
@@ -320,7 +320,7 @@ toBigNat (ByteArray arr) = BN# arr
 
 -- | Execute the extended Euclidean algorithm.
 -- For polynomials @a@ and @b@, compute their unique greatest common divisor @g@
--- and the unique coefficient polynomial @s@ satisfying @as + bt = g@.
+-- and the unique coefficient polynomial @s@ satisfying @a * s + b * t = g@.
```

Comment:
That's probably a good opportunity to use LaTeX.

Reply:
I replaced this with `a \cdot s + b \cdot t = g`, is that good?